src/store/types.ts

```diff
@@ -0,0 +1,2 @@
+export const PUT_PET :string = 'PUT_PET';
+export const LOAD_PET :string = 'LOAD_PET';
```

Comment:
The `: string` type is not required during const assignment;

Not perfect
```
const ten: number = 10;
const user: string = 'User';
```

Perfect
```
const ten = 10;
const user = 'User';
```

Reply:
corrected